lib/ui/pages/account-linking/account_discovery.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:pispapp/controllers/ephemeral/account-linking/account_discovery_controller.dart';
+import 'package:pispapp/controllers/ephemeral/account-linking/available_fsp_controller.dart';
+import 'package:pispapp/controllers/flow/account_linking_flow_controller.dart';
+import 'package:pispapp/models/account.dart';
+import 'package:pispapp/repositories/firebase/consent_repository.dart';
+import 'package:pispapp/ui/theme/light_theme.dart';
+import 'package:pispapp/ui/widgets/bottom_button.dart';
+import 'package:pispapp/ui/widgets/moja_button.dart';
+import 'package:pispapp/ui/widgets/shadow_box.dart';
+import 'package:pispapp/ui/widgets/title_text.dart';
+
+class AccountDiscovery extends StatelessWidget {
+  AccountDiscovery(this.fspId, this.fspName);
+
+  final String fspId;
+  final String fspName;
+  final AccountDiscoveryController _accountDiscoveryController =
+      AccountDiscoveryController();
+  final AccountLinkingFlowController _accountLinkingFlowController =
+      AccountLinkingFlowController(ConsentRepository());
+
+  Widget _buildActionSection() {
+    return GetBuilder<AccountLinkingFlowController>(
+      init: _accountLinkingFlowController,
+      global: false,
+      builder: (controller) {
+        if (!controller.isAwaitingUpdate) {
+          // The user is only allowed to click the button once.
+          // Afterward, the state of the payment flow controller will be updated
+          // to be awaiting for response. Once the response is received, the
+          // flow controller is expected to bring used to another page.
```

Comment:
```suggestion
          // flow controller is expected to bring user to another page.
```

Does this typo occur for the transfer screen as well? lol sorry i didn't notice if it does

Reply:
Probably haha, I took that part of the code from the transfer :D